test/util/test_runners/integrations_tests_runner.py

```diff
@@ -0,0 +1,138 @@
+"""
+This is a runner module for all integration tests.
+The purpose of this file is to be a cross platform tests runner
+for all integration tests
+
+Roadmap:
+1) run all integration tests from this file
```

Comment:
Shouldn't 1 and 2 be reversed? We need to setup the system env before running the tests no?